docs/_embed/quick-account-creation.md

```diff
@@ -59,7 +63,26 @@ https://zapier.com/webintent/create-zap?template=<zap-template-id>&utm_source=pa
 | **sign-up-email**       |  Required   | Email address of the user signing up.                                                              |
 | **refer** or **referrer**|  Required   | URL of the page where the user clicked the link.                                                   |
 
-After an account is created, a [user token still needs to be procured](https://platform.zapier.com/embed/partner-api#access-token) to access specific Partner API endpoints. Generally, since the user will already be signed in to their newly created account in an active session on Zapier, users won't have to explicitly sign in again when prompted with Zapier's OAuth flow.
+### Procure a token and redirect users to a custom URL
+
+This implementation [provides an access token](https://platform.zapier.com/embed/partner-api#procuring-a-token) to [Zapier's Partner API](https://platform.zapier.com/embed/partner-api), then allows you to redirect users to a custom URL of your choosing.
```

Comment:
Just adding a little more to this to include the steps of how this works

This implementation allows a new user accounts to be created, [provide an access tokens](https://platform.zapier.com/embed/partner-api#procuring-a-token) to [Zapier's Partner API](https://platform.zapier.com/embed/partner-api), and redirect users to a custom URL of your choosing.

Reply:
TY!